definitions/features/candlepin_database.rb

```diff
@@ -19,9 +19,10 @@ def validate_available_in_cpdb?
     check_option_using_cpdb_help('validate')
   end
 
-  def check_option_using_cpdb_help(option_name, parent_cmd = '')
+  def check_option_using_cpdb_help(option_name, followed_by_whitespace = true, parent_cmd = '')
     parent_cmd = '/usr/share/candlepin/cpdb' if parent_cmd.empty?
-    help_cmd = "#{parent_cmd} --help |  grep -c '\\-\\-#{option_name} '"
+    option_name += ' ' if followed_by_whitespace
```

Comment:
What is the reason for this? Wouldn't matching for whole word via `grep -c '\\-\\-#{option_name}\b'"` solve it or it's something else?

Reply:
@iNecas,
It is matching whole word. 
I did mistake while testing command on CLI. Extra-space not needed so removed.